src/global/pid_lut/pid_lut.cc

```diff
@@ -23,6 +26,42 @@ void InitPlugin(JApplication* app) {
 
   using namespace eicrecon;
 
+  //-------------------------------------------------------------------------
+  // FarBackward PID Through Phase Space
+  //-------------------------------------------------------------------------
+  PhaseSpacePIDConfig phase_space_pid_cfg{
+      .system        = "TaggerTracker_ID",
+      .direction     = {0.0, 0.0, -1.0},  // Direction is along z-axis
+      .opening_angle = 12 * dd4hep::mrad, // Beampipe opening angle
+      .pdg_value     = 11,                // Set PID to electron
+  };
+
+  app->Add(new JOmniFactoryGeneratorT<PhaseSpacePID_factory>(
+      "FarBackwardTruthSeededPhaseSpacePID",
+      {
+          "ReconstructedTruthSeededChargedWithoutPIDParticles",
+          "ReconstructedTruthSeededChargedWithoutPIDParticleAssociations",
+      },
+      {
+          "ReconstructedTruthSeededChargedWithFBPIDParticles",
+          "ReconstructedTruthSeededChargedWithFBPIDParticleAssociations",
+          "FarBackwardPhaseSpaceParticleIDs",
```

Comment:
You need to add those to output PODIO processor list of collections to prevent dangling references.